package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "teamwilder",
+  "name": "nextdoornature",
```

Comment:
I think this is technically the repo name which is teamwilder, but willing to be wrong on that.

Reply:
It's meant to be the name of the package, like if you publish on npm - so it should match the repo but it won't affect anything,